getting-started/generative-ai.rst

```diff
@@ -14,12 +14,12 @@ Acceptable uses
 
 Some of the acceptable uses of generative AI include:
 
-- Assistance with writing comments, especially in a non-native Language
+- Assistance with writing comments, especially in a non-native language
 - Gaining understanding of existing code
 - Supplementing contributor knowledge for code, tests, and documentation
 
 Unacceptable uses
 =================
 
 If a maintainer believes that a PR or issue was **fully generated** by AI tools, the PR or issue will be closed.
-If a contributor opens multiple, fully generated AI issues or PRs, the contributor may be blocked.
+If a contributor opens multiple fully AI-generated issues or PRs, the contributor may be blocked.
```

Comment:
Maybe something like:

"Maintainers may close issues and PRs that are not useful or productive, including
those that are fully generated by AI. If a contributor repeatedly opens unproductive
issues or PRs, they may be blocked."

(Can't provide a suggestion to replace both lines at once.)

Reply:
Adding this suggestion. Thanks!